src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -1,15 +1,29 @@
 ﻿using System;
 using CoreGraphics;
 using Foundation;
+using Microsoft.Maui.Devices;
 using Microsoft.Maui.Graphics;
-using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class EditorHandler : ViewHandler<IEditor, MauiTextView>
 	{
-		protected override MauiTextView CreatePlatformView() => new MauiTextView();
+		protected override MauiTextView CreatePlatformView()
+		{
+			var platformEditor = new MauiTextView();
+
+			if (DeviceInfo.Idiom == DeviceIdiom.Phone)
```

Comment:
@jsuarezruiz do we need to check the Idiom everywhere in the code that we use this `InputAccessoryView` ?

Reply:
We want this behavior on iOS, not Desktop but we don't need the same check everywhere (TimePicker etc).